### PR TITLE
Week 1 Basic Changes

### PR DESCRIPTION
**Pull request by rqtrinh**
Add image, bio, CSS styling, and random greeting. Week 1 tutorial up to step 11/14.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,7 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Ryan Trinh</h1>
+      <img src ="/images/selfie.jpg" width="300" height="300" class="center"/>
```
Comment (RyanSCollins) [inline portfolio/src/main/webapp/index.html]: Nit: Add alt text describing the image for accessibility.
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Comment (RyanSCollins) [inline portfolio/src/main/webapp/index.html]: Try to keep presentational concerns like width/height and centering in the stylesheet and have the html file contain only content.
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

**Review by RyanSCollins — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (rqtrinh) [reply portfolio/src/main/webapp/index.html]: Added an id "selfie" to describe the image for accessibility in new commit

Comment (rqtrinh) [reply portfolio/src/main/webapp/index.html]: Added an id "selfie", used this id to style width/height in the CSS stylesheet in new commit

**Review by RyanSCollins — APPROVED** [review]

**Pull request merged**